File: Java/Numbers/UnitConverter/UnitConverter.java
import java.util.Scanner;

/**
 * Unit Converter (temp, currency, volume, mass and more) – Converts various units between one another.
 * The user enters the type of unit being entered, the type of unit they want to convert to
 * and then the value. The program will then make the conversion.
 * Created by sraone on 10/5/14.
 */
public class UnitConverter {
    // Main Method
    public static void main(String[] args) {
        Scanner S = new Scanner(System.in );
        System.out.print("Conversions available ... \n\t" +
                "1. Temperature\n\t" +
                "2. Memory\n\t" +
                "3. Distance\n" +
                "Please select a conversion ... ");
        int conversion = S.nextInt();
        switch (conversion) {
            case 1 :
                Temperature.convert();
                break;
            case 2 :
                Memory.convert();
                break;
            case 3:
                Distance.convert();
                break;
            default:
                System.out.println("No such conversion available ... \nExiting ... \n");
        }
    }
}

File: Python/Numbers/UnitConverter/UnitConverter.py
'''
Created on 11-Oct-2014

@author: ghantasa
'''

from Temperature import Temperature
from Distance import Distance
from Memory import Memory
from Weight import Weight

if __name__ == '__main__':
    while (True):
        choice = input ('Available conversions ... \n' +
                    '1. Temperature\n' +
                    '2. Distance\n' +
                    '3. Memory\n' +
                    '4. Weight\n' +
                    '5. Exit\n' +
                    'Please enter your choice ... ' )
        if choice == '1' :
            t = Temperature()
            t.convert()
        elif choice == '2' :
            d = Distance()
            d.convert()
        elif choice == '3':
            m = Memory()
            m.convert()
        elif choice == '4' :
            w = Weight()
            w.convert()
        else:
            print('Exiting now ... Bye!')
            break
    